main/src/library/Environment.flix

```diff
@@ -28,15 +31,15 @@ mod Environment {
     /// Returns an map of the current system environment.
     ///
     pub def getEnv(): Map[String, String] = region rc {
-        import static java.lang.System.getenv(): ##java.util.Map \ rc;
-        import java.util.Map.entrySet(): ##java.util.Set \ { rc };
-        import java.util.Set.iterator(): ##java.util.Iterator \ { rc };
+        import static java.lang.System.getenv(): JMap \ rc;
+        import java.util.Map.entrySet(): JSet \ { rc };
+        import java.util.Set.iterator(): JIterator \ { rc };
         let _ = rc; // Avoids redundancy error.
         try {
             let iter = getenv() |> entrySet |> iterator;
             getEnvHelper(iter, Map.empty())
         } catch {
-            case _:##java.lang.Exception => Map.empty()
+            case _: ##java.lang.Exception => Map.empty()
```

Comment:
The try-catch rule just looks up the java class in `Resolver` and does not consider imports. Should I open a ticket, or is this going to be fixed?

Reply:
The solution is just to first do a lookup in the imports to get the actual java class and if that fails then fall back to trying to find a class for the literal (which is the current impl). Maybe the other way around?

Reply:
It seems this should be fixed in namer or maybe weeder even?

Reply:
@mlutze Any insights?

@jaschdoc If you can fix it, probably in Resolver, you are welcome.

Reply:
I'll take a stab at it in a new pr

Reply:
I would guess that this is parsed specifically as a Java type, so you'll might need changes from parser to resolver.